demisto_sdk/commands/format/update_generic.py

```diff
@@ -463,3 +463,37 @@ def check_server_version(self):
                 else:
                     self.data[self.from_version_key] = current_from_server_version
                     self.data.pop(self.json_from_server_version_key)
+
+    def adds_period_to_description(self):
+        """Adds a period to the end of the descriptions
+        if it does not already end with a period."""
+        logger.info("Adds a period to the end of the description")
+
+        def _add_period(value: Optional[str]) -> Optional[str]:
+            if (
+                value
+                and isinstance(value, str)
+                and not value.endswith(".")
+                and len(value.split(" ")) >= 3
```

Comment:
@shmuel44 why the check for >=3 words? 
can we add an explanation for this exclusion
@dorschw FYI